requirements.in

```diff
@@ -1,35 +1,35 @@
 # The file contains the direct ckan requirements.
 # Use pip-compile to create a requirements.txt file from this
 Babel==2.3.4
-Jinja2==2.8
-Pylons==0.9.7
 Beaker==1.7.0 # need to pin this because of https://github.com/bbangert/beaker/commit/fc511ceda4305fcf54919b3f081fed7790e2fef3
-bleach==1.4.2
-WebTest==1.4.3  # need to pin this so that Pylons does not install a newer version that conflicts with WebOb==1.0.8
+bleach==1.4.3
 fanstatic==0.12
-Markdown==2.4
-ofs==0.4.1
+Flask==0.10.1
+Jinja2==2.8
+Markdown==2.6.6
+ofs==0.4.2
 ordereddict==1.1
-Pairtree==0.7.1-T
+Pairtree==0.5.8
```

Comment:
True, Requires.io marks 0.5.8 as the latest, the "T" confused me, I think that it was a Beta release or something. I send comit with the two fixes